src/ado2gh/Commands/DownloadLogsCommand.cs

```diff
@@ -60,6 +61,12 @@ HttpDownloadService httpDownloadService
                 Description = "Local file to write migration log to (default: migration-log-ORG-REPO.log)."
             };
 
+            var wait = new Option("--wait")
+            {
+                IsRequired = false,
+                Description = "Waits for log to be ready to download."
+            };
```

Comment:
Based on the live feedback it'd be better to get rid of `--wait` and always wait with a timeout.

Reply:
Added in ed1afe023dff7250306495b522a6a64db736c407!